ExpressionEngine/ValueContainer.cs

```diff
@@ -24,7 +24,7 @@ public ValueContainer(string value, bool tryToParse = false)
 
             if (tryToParse)
             {
-                if (value.Contains('.') && double.TryParse(value, out var fValue))
+                if (value.Contains('.') && decimal.TryParse(value, out var fValue))
```

Comment:
Ikke specifik til denne PR, men . er vel locale afhænging. Bør try parse ikke selv håndtere om den indeholde .?

Reply:
Nej, "sproget" kan kun håndtere decimaltal med `.`. Det giver mening hvis vi f.eks. har: `add(1,1,1)`, er det så `1 + 1 + 1` eller `1,1 + 1`

Reply:
men value container, der kan man jo instancere den med både "10,0" og "10.", og på samme måde i expressions kan man vel også escape sig ud af det så man ender med danske "10,0" ?

Reply:
Hmm. Hvordan tænker du sådan en expression kommer til at se ud?

Reply:
Hvis man laver `add(float('23,3'), float('1,7'))` vil give `25`, men `float(2,3)` vil give en fejl, da `float` kun tager en parameter som er string og ikke to integers :D